src/tree_schema.h

```diff
@@ -1898,30 +1898,38 @@ LY_ERR lysc_iffeature_value(const struct lysc_iffeature *iff);
  */
 LY_ERR lysc_feature_value(const struct lysc_feature *feature);
 
-#define LYXP_SCNODE 0x02        /**< No special tree access modifiers. */
-#define LYXP_SCNODE_SCHEMA 0x04 /**< Apply node access restrictions defined for 'when' and 'must' evaluation. */
-#define LYXP_SCNODE_OUTPUT 0x08 /**< Search RPC/action output nodes instead of input ones. */
+/**
+ * @defgroup findxpathoptions Atomize XPath options
+ * Options to modify behavior of ::lys_find_xpath() and ::lys_find_xpath_atoms() searching for schema nodes in schema tree.
+ * @{
+ */
+#define LYS_FIND_XP 0x02        /**< Default value, searching in schema tree with no special access modifiers. */
```

Comment:
This flag does not have to be public at all, `LYXP_SCNODE_SCHEMA` as an internal value is enough, this can be removed.

Reply:
ok, removed